tests/files/mpeg.rs

```diff
@@ -306,3 +307,57 @@ fn remove_id3v1() {
 fn remove_ape() {
 	crate::remove_tag!("tests/files/assets/minimal/full_test.mp3", TagType::Ape);
 }
+
+#[test]
+fn read_and_write_tpil_frame() {
+	let key_value_pairs = vec![
+					("engineer".to_string(), "testperson".to_string()), 
+					("vocalist".to_string(), "testhuman".to_string())
+				];
+
+	let mut file = temp_file!("tests/files/assets/issue_213.mp3");
```

Comment:
This file doesn't exist, did you forget to commit it?

Reply:
Please don't add more binary files to the repo!

This is not needed for these kind of round-trip tests. It is easier to generate the expected contents of the file programmatically. That's why I proposed to first write and then (re-)read the written data.

Reply:
Here you find some examples: https://github.com/Serial-ATA/lofty-rs/blob/f1e8ab6c2fb1bb275a15d9e4246ad49f2dbbb3da/tests/files/mpeg.rs#L250